src/acomms/modemdriver/popoto_driver.cpp

```diff
@@ -268,74 +287,65 @@ void goby::acomms::popotoDriver::send_range_request(int dest)
 
     // Send over the wire
     signal_and_write(range.str());
-
 }
 
 // --------------------------- Incoming msgs ------------------------------------------------
-void goby::acomms::popotoDriver::do_work()
+void goby::acomms::PopotoDriver::do_work()
 {
     std::string in;
     while (modem_read(&in))
     {
         try
         {
-            // let others know about the raw feed. Can remove this if not needed
-            protobuf::ModemRaw raw;
-            raw.set_raw(in);
-            ModemDriverBase::signal_raw_incoming(raw);
-
             // Remove VT100 sequences (if they exist) and popoto prompt
-            std::cerr << "This is then recieved string: " << in << std::endl;
             in = StripString(in, "Popoto->");
+            constexpr const char *VT100_BOLD_ON = "\x1b[1m", *VT100_BOLD_OFF = "\x1b[0m";
```

Comment:
Using #define in C++ is generally a bad idea since the preprocessor is a pretty blunt tool. constexpr is better in C++11 and newer.